includes/class-wc-siftscience-admin.php

```diff
@@ -141,29 +145,36 @@ public function output_settings_fields() {
 
 		/**
 		 * Outputs the main settings page
+		 * Creates the HTML table for the batch upload control
 		 */
 		private function output_settings_main() {
 			WC_Admin_Settings::output_fields( $this->get_settings() );
 
-			echo $this->styling_checkbox_label( WC_SiftScience_Options::AUTO_SEND_ENABLED );
-			echo $this->batch_upload();
-			$data = array( 'api' => admin_url( 'admin-ajax.php' ) );
+			$this->styling_checkbox_label( WC_SiftScience_Options::AUTO_SEND_ENABLED );
+
+			echo <<<'table'
```

Comment:
Interesting. Why did you move the the code in this case, but not for `styling_checkbox_label`?

Reply:
Also, this needs `wp_kses` :-)

Reply:
label code was used twice unlike batch control

Reply:
strangely I was not prompt